API_changes.rst

```diff
@@ -2,6 +2,10 @@ API changes
 ===========
 Versions (X.Y.Z) where Z > 0 e.g. 3.0.1 do NOT have API changes!
 
+API changes 3.8.7
+-----------------
+- Drop support for asyncio task cancellation
```

Comment:
This is not correct, it was a problem earlier see your own callback problem.

There are no API changes.